src/executionFiles/bridges/anyswap.execute.ts

```diff
@@ -130,8 +135,8 @@ export class AnySwapExecutionManager {
     waitForTxProcess.txLink =
       toChain.metamask.blockExplorerUrls[0] + 'tx/' + waitForTxProcess.txHash
     waitForTxProcess.message = 'Funds Received:'
-    status.fromAmount = parsedReceipt.fromAmount
-    status.toAmount = parsedReceipt.toAmount
+    // status.fromAmount = parsedReceipt.fromAmount
+    // status.toAmount = parsedReceipt.toAmount
```

Comment:
Was this removed on purpose? Please document why this is commented out 😇

Reply:
Oh sorry, the "comment" is only in the commit. There is actually no real parsing of the receipt build for anyswap yet, so these amounts are always `'0'` which makes following requests fail.

Reply:
Ah alright 👍